examples/set-default-labels-nested-package/README.md

```diff
@@ -0,0 +1,39 @@
+# set-default-labels: Nested Package Example
+
+### Overview
+
+This example shows how to run [`set-default-labels`] in a nested KPT package. 
+
+### Fetch the example package
+
+Get the example package by running the following commands:
+
+```shell
+kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/set-default-labels-nested-package
+```
+
+Take a look at the `Kptfile`, it has the `set-default-labels` configured. Note, it does not require function configs (no `configPath` nor `configMap`).
```

Comment:
A few grammar nits

> Note,

Note:

> no `configPath` nor `configMap`

This reads odd. I suspect it may not be the right grammar. Please rephrase it.

Reply:
indeed. Done